src/pkg/db/dependency.go

```diff
@@ -124,13 +124,14 @@ func DependencyRequestToFilters(req *terrariumpb.ListDependenciesRequest) []Filt
 	return filters
 }
 
-var results []DependencyResult
-
 func (db *gDB) Fetchdeps() []DependencyResult {
-	db.g().Model(&Dependency{}).
-		Select("dependencies.id AS DependencyID, dependencies.interface_id AS InterfaceID").
-		Joins("INNER JOIN dependency_attributes ON dependencies.id = dependency_attributes.dependency_id").
-		Scan(&results)
+	var results []DependencyResult
+
+	db.g().Raw(`
+        SELECT dependencies.id AS "DependencyID", dependencies.interface_id AS "InterfaceID"
+        FROM dependencies
+        INNER JOIN dependency_attributes ON dependencies.id = dependency_attributes.dependency_id
+    `).Scan(&results)
 
 	return results
 }
```

Comment:
Since this fixes a blocking issue, we may let proceed by merging this, but for future:

We should’t ignore error here. And consider filtering based on requirement instead of returning the entire table in one response.

For doing this, we can use the existing `QueryDependencies` function with relevant options. Reducing duplicate functionality should help in maintenance overhead.